### DISCUSSION TITLE
Contribuito allo sviluppo di Giua

### DISCUSSION THREAD

**Discussion by robertopadovesi** [iisgiua/giuaschool#785] [Idee e proposte]
Buongiorno @trinko,

ho deciso di cercare di fare il mio piccolo, con le conoscenze che ho, per migliorare il registro e aiutarti in questo magnifico progetto! pertanto, volevo chiederti come potevo aiutarti e in che maniera potevo farlo.

Grazie!

**Comment by robertopadovesi** [comment]
Nel caso ''provassi'' a fare questa cosa, potrei, provare, per primo, a fare la funzionalità di adozione dei libri di testo? Sto facendo delle esercitazioni e ne avrei uno pronto, dimmi tu.

**Reply by trinko** [reply]
Certo, provare a sviluppare questa funzionalità sarebbe senz'altro utile.
